website/promotion/templates/requests/new_request_email.txt

```diff
@@ -0,0 +1,11 @@
+{% load i18n %}{% blocktrans with request_event_title=request.event.title request_publish_date=request.publish_date request_channel=request.channel%}Hi Paparazcie,
```

Comment:
I don't think this `with (...)` is required here as it's not doing anything special